src/repl.ts

```diff
@@ -309,6 +331,15 @@ export function repl(prompt: string) {
     if (/^:paste/.test(code) && !multilineBuffer) {
       return enterPasteMode()
     }
+    if (/^:load/.test(code) && !multilineBuffer) {
+      let files = (code.match(/\S+/g) || []).slice(1);
```

Comment:
what if there is a space in the filename? I think `load` command can load only one file at one time